pkg/resources/privileges.go

```diff
@@ -1,90 +1,77 @@
 package resources
 
-type privilege string
+type Privilege string
```

Comment:
made these public for a follow-up refactor I'm planning, apologies for the noise